tests/test_procedure.py

```diff
@@ -0,0 +1,50 @@
+import os
+import tempfile
+from unittest import mock
+from unittest.mock import MagicMock
+from zipfile import ZipFile
+
+import pytest
+
+import snowcli.utils
+from snowcli.cli.snowpark import procedure
+from snowcli.utils import SplitRequirements
+
+
+class TestProcedure:
+    TEMP_DIRECTORY_NAME = "test_procedure_tmp"
+    DIR_INITIAL_CONTENTS = {
+        "requirements.txt",
+        "local_connection.py",
+        ".gitignore",
+        "app.py",
+        "config.toml",
+    }
+
+    def test_procedure_init(self, tmp_dir_for_procedure_tests):
+        procedure.procedure_init()
```

Comment:
I think in other unit tests we use `runner` to invoke the command instead calling the method directly. I think we should decided witch approach we want to go.

Reply:
Done